src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/SmokeTestHelperResourceIntTest.kt

```diff
@@ -119,7 +123,12 @@ class SmokeTestHelperResourceIntTest : ResourceTest() {
 
     @Test
     fun `will change prisoner name`() {
-      val prisonerNo = createPrisoner()
+      var prisonerNo: String? = null
```

Comment:
bit naff here, but the `builder.build` doesn't return anything

Reply:
I think that makes sense, since what would it return? Especially if you created lots of prisoners?